lms/templates/error.html.jinja2

```diff
@@ -3,8 +3,10 @@
 {% set title = message %}
 
 {% block content %}
-  <img src="{{ 'lti:static/images/sad-annotation.svg'|static_path }}">
-  <p>
+  <div>
+    <img src="{{ 'lms:static/images/sad-annotation.svg'|static_path }}">
+  </div>
+  <div>
     {{ message|safe }}
-  </p>
+  </div>
```

Comment:
Why add the `<div>` around the `<img>`? That seems to be unnecessary.

And why change the `<p>` around the text to a `<div>`? Also seems unnecessary as far as I can tell.

It seems to work fine (and is simpler and more semantically correct) how it was before?

```html
<img src="{{ 'lms:static/images/sad-annotation.svg'|static_path }}">
<p>{{ message|safe }}</p>
```